onecgiar-pr-client/src/app/shared/services/export-tables.service.ts

```diff
@@ -109,26 +109,89 @@ export class ExportTablesService {
     }
   }
 
-  exportMultipleSheetsExcel(list: any[], fileName: string, wscols?: Wscols[], tocToExport?: any[], callback?) {
+  async exportMultipleSheetsExcel(list: any[], fileName: string, wscolsResults?: any[], tocToExport?: any[], wscolsToc?: any[], callback?) {
     try {
-      import('xlsx').then(xlsx => {
-        const worksheet = xlsx.utils.json_to_sheet(list, { skipHeader: Boolean(wscols?.length) });
-        const tocsheet = xlsx.utils.json_to_sheet(tocToExport, { skipHeader: Boolean(wscols?.length) });
-        if (wscols) {
-          worksheet['!cols'] = wscols as any;
-          tocsheet['!cols'] = wscols as any;
+      import('exceljs').then(async ExcelJS => {
```

Comment:
@xKeCo isn't there a better way to do this? I don't think we should be working with the imports the way it is being done here.

Reply:
@gcmartinezg This import is a valid way to perform a dynamic import of the library because it is only imported when the user needs it.

Reply:
Understood